compiler+runtime/test/bash/clojure-test-suite/deps.edn

```diff
@@ -0,0 +1,4 @@
+{:paths ["src"]
+ :deps {io.github.jank-lang/clojure-test {;; TODO update to official repo
+                                          :git/url "https://github.com/frenchy64/clojure-test.git"
+                                          :git/sha "d5c1a319251b25e3afd3acd07648c3950f882fa3"}}}
```

Comment:
We're using submodules for everything right now. I'd prefer to stick with that. Let's add the submodule here. I've also renamed that repo to clojure-test-suite, so we'll have `clojure-test-suite/clojure-test-suite`, but that's ok.

Reply:
Done.